clicommand/global.go

```diff
@@ -48,10 +48,17 @@ var NoHTTP2Flag = cli.BoolFlag{
 
 var DebugFlag = cli.BoolFlag{
 	Name:   "debug",
-	Usage:  "Enable debug mode",
+	Usage:  "Enable debug mode. Synonym for `--log-level debug`. Takes precedence over `--log-level`",
 	EnvVar: "BUILDKITE_AGENT_DEBUG",
 }
 
+var LogLevelFlag = cli.StringFlag{
+	Name:   "log-level",
+	Value:  "notice",
+	Usage:  "Set the log level for the agent, making logging more or less verbose. Defaults to notice",
+	EnvVar: "BUILDKITE_LOG_LEVEL",
```

Comment:
The rest of these have `BUILDKITE_AGENT_` prefix — was that intentional? Do you see this as broader than just the agent?

Reply:
😶 you're right, i'll fix this up